packages/@react-spectrum/s2/src/SelectBoxGroup.tsx

```diff
@@ -0,0 +1,451 @@
+  },
+  overflow: 'hidden',
+  textAlign: {
+    default: 'center',
+    orientation: {
+      horizontal: 'start'
+    }
+  },
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  }
+});
+
+const labelText = style({
+  gridArea: 'label',
+  alignSelf: 'center',
+  justifySelf: {
+    default: 'center',
+    orientation: {
+      horizontal: 'start'
+    }
+  },
+  width: '100%',
+  overflow: 'hidden',
+  minWidth: 0,
+  textAlign: {
+    default: 'center',
+    orientation: {
+      horizontal: 'start'
+    }
+  },
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  fontWeight: {
+    orientation: {
+      horizontal: 'bold'
+    }
+  },
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  }
+});
+
+const gridStyles = style({
+  display: 'grid',
+  gridAutoRows: '1fr',
+  gap: {
+    gutterWidth: {
+      default: 16,
+      compact: 8,
+      spacious: 24
+    }
+  }
+}, getAllowedOverrides());
+
+/**
+ * SelectBox is a single selectable item in a SelectBoxGroup.
+ */
+export function SelectBox(props: SelectBoxProps): ReactNode {
+  let {children, value, isDisabled: individualDisabled = false, UNSAFE_style} = props;
+  
+  let {
+    orientation = 'vertical',
+    isDisabled: groupDisabled = false,
+    showCheckbox = false
+  } = useContext(SelectBoxContext);
+
+  const size = 'M';
+  const isDisabled = individualDisabled || groupDisabled;
+
+  return (
+    <ListBoxItem
+      id={value}
+      isDisabled={isDisabled}
+      textValue={value}
+      className={renderProps => (props.UNSAFE_className || '') + selectBoxStyles({
+        size, 
+        orientation, 
+        isDisabled: renderProps.isDisabled,
+        isSelected: renderProps.isSelected,
+        isHovered: renderProps.isHovered,
+        isFocusVisible: renderProps.isFocusVisible
+      }, props.styles)}
+      style={UNSAFE_style}>
+      {(renderProps) => (
+        <>
+          {showCheckbox && (renderProps.isSelected || (!renderProps.isDisabled && renderProps.isHovered)) && (
+            <div 
+              className={style({
+                position: 'absolute',
+                top: 12,
+                left: 12
+              })}
+              aria-hidden="true">
+              <div
+                className={box({
+                  isSelected: renderProps.isSelected,
+                  isDisabled: renderProps.isDisabled,
+                  size: 'M'
+                } as any)}>
+                {renderProps.isSelected && (
+                  <Checkmark 
+                    size="S" 
+                    className={iconStyles} />
+                )}
+              </div>
+            </div>
+          )}
+          <Provider
+            values={[
+              [IllustrationContext, {
+                size: 'S',
+                styles: illustrationContainer({size: 'S', orientation, isDisabled: renderProps.isDisabled})
+              }],
+              [TextContext, {
+                slots: {
+                  label: {
```

Comment:
add the default slot as a possibility for the label as well. see Menu for an example https://github.com/adobe/react-spectrum/blob/a8dbaf6907ef770bc5ebb28a2f110b9eb6c5e4e0/packages/%40react-spectrum/s2/src/Menu.tsx#L495